dapps/organizations/src/components/detail/identifications/notary/request/request.vue

```diff
@@ -275,12 +275,17 @@
 
         <!-- start button -->
          <template v-if="status === -1">
-          <button type="button" class="btn btn-primary btn-rounded mx-auto"
+
+          <button v-if="enoughFunds" type="button" class="btn btn-primary btn-rounded mx-auto"
             id="ident-request-start"
             :disabled="(status === 1 && !requestForm.isValid) || sending"
             @click="status += 1">
             {{ `_org.ident.notary.request.request-verification` | translate }}
           </button>
+
+          <p class="w-100 text-center text-danger" v-if="!enoughFunds">
+            {{ $t(`_org.ident.notary.request.not-enough-funds`,{readableFunds:readableFunds}) }}
```

Comment:
is it also possible to use es6 syntax {readableFunds}  ?

Reply:
done